vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -44,15 +53,51 @@ var (
 		prometheus.CounterOpts{
 			Namespace: metricsNamespace,
 			Name:      "usage_sample_missing_recommendation_count",
-			Help:      "Count of usage samples when a recommendation should be present but is missing.",
+			Help:      "Count of usage samples when a recommendation should be present but is missing",
 		}, []string{"resource", "is_oom"},
 	)
+	cpuUsageRecommendationDiff = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Name:      "cpu_usage_recommendation_diffs_cores",
+			Help:      "Absolute diffs between usage and recommendation for cpu",
```

Comment:
nit: s/cpu/CPU/ - to make it consistent with cpu_recommendation_cores

Reply:
Done.